code/game/objects/items/devices/scanners.dm

```diff
@@ -397,7 +397,7 @@ REAGENT SCANNER
 	if(!user.incapacitated() && Adjacent(user))
 
 		if(cooldown)
-			to_chat(user, "<span class='warning'>[src]'s barometer function is preparing itself.</span>")
+			to_chat(user, "<span class='warning'>[src]'s barometer function is prepraring itself.</span>")
```

Comment:
Same here

Reply:
Rebased branch based on an updated one with the same changes (I think)